pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -335,6 +336,8 @@ func (exp *flowExporter) sendDataRecord(dataRec ipfix.IPFIXRecord, record flowex
 				// this dummy IP address.
 				_, err = dataRec.AddInfoElement(ie, net.IP{0, 0, 0, 0})
 			}
+		case "destinationServicePort":
+			_, err = dataRec.AddInfoElement(ie, record.Conn.TupleOrig.DestinationPort)
```

Comment:
this is fine, but I believe this information can be retrieved from `destinationServicePortName` by the aggregator, although it should require access to the Service spec

Reply:
Yes it is possible. Currently there is no access to shared informers yet in flow-aggregator service yet.